app/config/config.yml

```diff
@@ -191,3 +191,13 @@ httplug:
         default:
             factory: httplug.factory.curl
             plugins: [httplug.plugin.logger]
+
+lexik_translation:
+    fallback_locale: [en]
+    managed_locales: [en]
+
+    base_layout: "PrimeTranslationsBundle::pagelayout.html.twig"
```

Comment:
Needs to be referenced with Twig notation: `@PrimeTranslations/pagelayout.html.twig`

Reply:
Fixed in https://github.com/netgen/media-site/pull/32/commits/30eaaea629b84d9c57c4b41c3f2831f7207b244d